debian/libsaimetadata-dev.install

```diff
@@ -1,3 +1,5 @@
 meta/sai*.h usr/include/sai
 meta/Sai*.h usr/include/sai
 SAI/meta/sai*.h usr/include/sai
+usr/lib/*/libsaimetadata.so
```

Comment:
why suddenly dev package contains so library? should that be in normal package? and dev should contain headers only?

Reply:
The normal package should just have the so library with the SONAME (i.e. `libsaimetadata.so.0`). The dev package should contain the headers as well as a symlink from the basic library name (`libsaimetadata.so`) to the actual so library (so a symlink from `libsaimetadata.so` to `libsaimetadata.so.0`). This line that's being added is a symlink.

See also https://github.com/sonic-net/sonic-sairedis/pull/1116#discussion_r1003510714